include/clap/ext/draft/webview.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+static CLAP_CONSTEXPR const char CLAP_EXT_WEBVIEW[] = "clap.webview/1";
+
+// clap.gui API constant. The pointer in clap_window must be NULL, but sizing methods are useful.
+// uses logical size, don't call clap_plugin_gui->set_scale()
+static const CLAP_CONSTEXPR char CLAP_WINDOW_API_WEBVIEW[] = "webview";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/// @page Webview
+///
+/// This extension enables the plugin to provide the start-page for a webview UI, and exchange
+/// messages back and forth.
+///
+/// Messages are received in the webview using a standard MessageEvent, with the data in an
+/// ArrayBuffer. They are posted back to the plugin using window.parent.postMessage(), with the
+/// data in an ArrayBuffer or TypedArray.
+
+typedef struct clap_plugin_webview {
+   // Returns the URL for the webview's initial navigation, as a null-terminated UTF-8 string.
+   // This must be called at least once before any messages are sent (or accepted by the host).
+   // If this URL is relative, it is resolved relative to the plugin (bundle) resource directory.
```

Comment:
The concept of bundle isn't yet defined within clap, at least for web clap we haven't specified it yet.
This isn't a blocker for the draft, but we'll have to define it before stabilizing this ext.